dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -37,7 +38,7 @@ def _wait_to_write(arr):
         raise RuntimeError("Can only wait for NDArray")
     mx.base._LIB.MXNDArrayWaitToWrite(arr.handle)
 
-def feed_ndarray(dali_tensor, arr):
+def feed_ndarray(dali_tensor, arr, cuda_stream = None):
```

Comment:
I would add some comment that in the MXNet in most cases None and 0 are sane. Other values doesn't make much sense unless you create this stream on your own somehow.

Reply:
done